chickadee/utils.py

```diff
@@ -30,16 +23,128 @@ def get_package(package):
         raise ProcessError(f"R package, {package}, is not installed")
 
 
-def set_end_date(end_date):
+def set_general_options(
```

Comment:
The transition from R to python in newly added utils' functions looks flawless to me.